crates/rolldown/tests/esbuild/dce/dce_of_iife/artifacts.snap

```diff
@@ -13,8 +13,7 @@ keepMe();
 var someVar;
 (([y]) => {})(someVar);
 (({ z }) => {})(someVar);
-var keepThis = /* @__PURE__ */ (() => stuff())();
-keepThis();
```

Comment:
The test seem to intent to keep it.

Reply:
The call is kept (`(/* @__PURE__ */ (() => stuff())())();`). It is inlined though.